app/views/govuk_publishing_components/components/docs/devolved_nations.yml

```diff
@@ -0,0 +1,77 @@
+name: Devolved Nations (experimental)
+description: A banner for linking to guidance for other nations.
+body: |
+  The component replaces uses of the important metadata component for guidance for other nations.
+
+  The component can display:
+
+  * nations that the guidance relates to
+  * a list of alternative URLs where applicable
+shared_accessibility_criteria:
+  - link
+accessibility_criteria: |
+  - Provide context for link text to highlight guidance is available.
```

Comment:
Nitpick: can you add `The component must:` above this line? Just to clarify and make it more readable.